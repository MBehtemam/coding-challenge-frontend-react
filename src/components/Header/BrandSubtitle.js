import styled from 'styled-components';

const BrandSubtitle = styled.h3`
  color: white;
  font-size: 1.5rem;
`;
export default BrandSubtitle;
