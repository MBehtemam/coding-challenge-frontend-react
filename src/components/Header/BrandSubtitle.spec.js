import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import BrandSubtitle from './BrandSubtitle';

describe('Test suits for brand subtitle', () => {
  it('should match with snapshot', () => {
    const tree = renderer.create(<BrandSubtitle />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
