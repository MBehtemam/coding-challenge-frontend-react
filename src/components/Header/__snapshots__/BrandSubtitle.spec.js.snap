// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suits for brand subtitle should match with snapshot 1`] = `
.c0 {
  color: white;
  font-size: 1.5rem;
}

<h3
  className="c0"
/>
`;
